gc/base/standard/SweepHeapSectioningSegmented.cpp

```diff
@@ -173,6 +176,9 @@ MM_SweepHeapSectioningSegmented::reassignChunks(MM_EnvironmentBase *env)
 				chunk->chunkBase = (void *)heapChunkBase;
 				chunk->chunkTop = (void *)heapChunkTop;
 				chunk->memoryPool = pool;
+				Assert_MM_true(NULL != pool);
+				chunk->_minFreeSize = OMR_MAX(pool->getMinimumFreeEntrySize(), pool->getSweepPoolManager()->getMinimumFreeSize());
```

Comment:
you can add a comment, like: "Some memory pools, like the one in LOA, may have larger min free then in the rest of the heap being swept"